package.json

```diff
@@ -2,12 +2,15 @@
   "name": "commons",
```

Comment:
Could you please explain why we have to have two package.jsons? I see that it is because we have the client folder, but don't quite understand why we need it. I haven't seen that before. Thanks!

Reply:
`creat-react-app` run internally on a separate server and also need to run it scripts and make a node environment. (it's like this server for the front)

`node-server` is the other one.

when u work on express and react ( actually u work on two servers)